docs/variables/countdown.md

```diff
@@ -31,11 +31,11 @@ This variable takes ***one*** *required* parameter that is a date string to coun
   * `02:00:00PM`
   * ...
 
-###### A list of more supported date and time formats can be found [**here**](https://github.com/araddon/dateparse)
+###### A list of more supported date and time formats can be found [**here**](https://github.com/araddon/dateparse).
 
 If you would like to specify a different timezone than coordinated universal time (UTC), you can do so by providing the UTC offset to that timezone (e.g. `GMT-0400` for EDT) at the end of the date string.
 
-###### A list of all existing timezones can be found [**here**](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)
+###### A list of all existing timezones can be found [**here**](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List).
```

Comment:
same as above with this

Reply:
Refer to [**this**](https://github.com/fossadev/fossabot-docs/pull/20#discussion_r788770539).